docs/index.asciidoc

```diff
@@ -90,29 +96,32 @@ Apply specific codecs for specific content types.
 The default codec will be applied only after this list is checked
 and no codec for the request's content-type is found
 
-[id="plugins-{type}s-{plugin}-host"]
-===== `host` 
+[id="plugins-{type}s-{plugin}-backlog"]
+===== `backlog` 
 
-  * Value type is <<string,string>>
-  * Default value is `"0.0.0.0"`
+  * Value type is <<number,number>>
+  * Default value is `128`
 
-The host or ip to bind
+Sets SO_BACKLOG for the receiving socket, which is the
+maximum queue length for incoming connection requests.
+If a connection request arrives when the queue is full,
+the connection is refused.
```

Comment:
Do we want to expose this an option ?  If I understand correctly this would only kick in for a HUGE burst of traffic where the payload is queued in the socket. If the pipeline is healthy the payload will get pulled from the socket ASAP and processed, if the pipeline is blocked the payload will get pulled from the socket ASAP and a 429 is returned.